### PR TITLE
Don't reset IPMI setting when update inventory

### PR DESCRIPTION
**Pull request by masa-orca**
##### SUMMARY

- Don't reset IPMI setting when update inventory
- Fixes https://github.com/ansible-collections/community.zabbix/issues/1110

##### ISSUE TYPE
- Bugfix Pull Request

##### COMPONENT NAME

zabbix_host module

### CODE REVIEW

File: changelogs/fragments/1162-do_not_update_ipmi_options.yml [hunk 1]
```diff
@@ -0,0 +1,2 @@
+bugfixes:
+  - zabbix_host - Don't reset IPMI setting when update inventory
```
Comment (BGmot) [inline changelogs/fragments/1162-do_not_update_ipmi_options.yml:2]: - zabbix_host - Don't reset IPMI setting when update inventory data of a host

**Comment by BGmot** [comment]
I don't understand why you treat let's say `ipmi_privilege` and `ipmi_username` differently? why some of them checked for being in `zabbix_host_obj` and some not?

**Review by BGmot — APPROVED** [review]

**Comment by masa-orca** [comment]
Thank you for reviewing. 

> `ipmi_privilege` and `ipmi_username` differently?

It is no meaning. I add checks about the parameters.

> why some of them checked for being in zabbix_host_obj and some not?

- We need to set the minimum parameters.
- `reseting IPMI setting` was coused by unnecessary parameters.
- In current source code, we cannot update value of string parameters (e.g. `ipmi_username`) with empty string.
  - To evaluate empty string result is false, so, the module skips adding parameter.
```python
test_string = ""
if test_string:
    print('Update a parameter')  # This print function is never called.
```

File: changelogs/fragments/1162-do_not_update_ipmi_options.yml (hunk 1 above)
Comment (masa-orca) [reply changelogs/fragments/1162-do_not_update_ipmi_options.yml]: Thanks. I have changed to the sentence which you wrote.

**Comment by BGmot** [comment]
Thank you very much @masa-orca -)

**Pull request merged**